src/mux-analytics.brs

```diff
@@ -788,23 +788,38 @@ function muxAnalytics() as Object
     if error <> Invalid
       if error.errorCode <> Invalid
         errorCode = error.errorCode
+      else if error.player_error_code <> Invalid
```

Comment:
I think we want to just simply use `if`s, that way the last one always wins, kind of in a waterfall.

So this would be essentially just replacing all your `else if`s with `if`s